CMakeLists.txt

```diff
@@ -13,10 +13,42 @@ TRIBITS_PACKAGE_DECL(DataTransferKit)
 TRIBITS_ADD_DEBUG_OPTION()
 TRIBITS_ADD_SHOW_DEPRECATED_WARNINGS_OPTION()
 
-TRIBITS_ADD_ETI_INSTANTIATIONS(TpetraCore
-  "S={double} LO={int} GO={unsigned long} N={Kokkos::Compat::KokkosSerialWrapperNode}"
-  )
+##---------------------------------------------------------------------------##
+## Explicit template instantiation
+##---------------------------------------------------------------------------##
+
```

Comment:
Add a comment that this code was taken from Ifpack2. This way later we can remember to sync if something changes.